Assets/MixedRealityToolkit-SDK/README.md

```diff
@@ -0,0 +1,3 @@
+# Mixed Reality Toolkit - SDK
```

Comment:
Weren't we going to have < Feature > folders in SDK as well?

Reply:
MixedRealityToolkit <- Pure C# (Mostly)
       /Required
       /< Features > <- Optional
   MixedRealityToolkit-SDK < Optiona l> <- Monobehaviours
       /< Features >

Reply:
The SDK folder can stay as it is for now until we start adding components.  A feature for a later sprint :D

Reply:
Agreed